CMSgov/mdct-mcr
# MDCT-2118: Question Change for D3.VIII.8

## Description

Closes [MDCT-2118](https://qmacbis.atlassian.net/browse/MDCT-2118).

Changing the format and verbiage of question `D3.VIII.8` under **Sanctions**.

### How to test

- Run the app locally
- Create a report
- Create a plan
- Navigate to `/mcpar/plan-level-indicators/sanctions`
- Add a sanction, check that the question is now a radio with an adorable little child
- Save it, make sure it's saved to the DB

### Changed Dependencies

N/A

## Code author checklist

## Reviewer checklist (two different people)
